### DISCUSSION TITLE
Disable Right=Accept Suggestion

### DISCUSSION THREAD

**Discussion by eggbean** [chrisant996/clink#523] [Q&A] [answered]
I just noticed that this message saying "Right=Accept Suggestion" comes up when the Fish-like autosuggestions comes up.

Is that new? I have not noticed it before and I've been using clink for years. I only noticed it as my terminal window was a lot smaller than usual.

Anyway, how do I disable this? I don't need to be reminded of that.

**Comment by chrisant996** [comment, marked as answer]
Yes it's new. So many people confuse suggestions versus completions, or don't realize the text isn't actually part of the command line yet.

The documentation section on auto suggestions says how to turn off the hint.

`clink set autosuggest.hint false`
